pkg/registry/apis/provisioning/resources/client.go

```diff
@@ -8,25 +8,43 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/dynamic"
 
 	dashboard "github.com/grafana/grafana/apps/dashboard/pkg/apis/dashboard/v1alpha1"
-	"github.com/grafana/grafana/pkg/apimachinery/utils"
 	folders "github.com/grafana/grafana/pkg/apis/folder/v0alpha1"
 	iam "github.com/grafana/grafana/pkg/apis/iam/v0alpha1"
 	"github.com/grafana/grafana/pkg/services/apiserver"
 	"github.com/grafana/grafana/pkg/services/apiserver/client"
 )
 
+var (
+	UserResource      = iam.UserResourceInfo.GroupVersionResource()
+	FolderResource    = folders.FolderResourceInfo.GroupVersionResource()
+	DashboardResource = dashboard.DashboardResourceInfo.GroupVersionResource()
+
+	// StandardResources is the list of resources that can fully managed from the UI
+	StandardResources = []schema.GroupVersionResource{FolderResource, DashboardResource}
+)
+
 type ClientFactory struct {
 	configProvider apiserver.RestConfigProvider
 }
 
+type ResourceClients interface {
```

Comment:
switched to interface so it is easier to know what should be exported -- in particular, the client supplier should not also be trying to process things